Lib/csv.py

```diff
@@ -416,6 +420,10 @@ def has_header(self, sample):
                 continue # skip rows that have irregular number of columns
 
             for col in list(columnTypes.keys()):
+                # checking if all col are strings
```

Comment:
```suggestion
                # check if all col are strings
```

Per standard programming convention, code comments are always written in the imperative